WTWSMS/common/minor_titles/WtWSMS_minor_titles.txt

```diff
@@ -132,32 +132,37 @@ title_sigecempa = {
 }
 
 title_court_celtic_poet = { # Celtic Poet/Bard
-	dignity = 0.25
+	dignity = 1
 	realm_in_name = yes
 	grant_limit = 1
 	opinion_effect = 30
 	
 	monthly_salary = 0.02
-	monthly_prestige = 0.01
+	monthly_prestige = 0.1
```

Comment:
> You are wrong and you have no reason to oppose my rectification of your crappy file. It is better, it is an improvement, and it solves the disgusting titles. Your file was utter crap. Your mercenaries sucked. They were terrible. You have no reason to not implement this change. I did your service.

No but seriously, drop the histrionic obsession. I'm flattered that the entire mod became mine suddenly... Rather than making all of this drama, prove why this change is justified? Why do you hate these titles all of a sudden?

> You are also a monstrous hypocrite: remember when you forced me to change "DONAMONAPRISC_DECADENT" because it "isn't relevant" or connected to the Modifer's purpose?

A golden rule when coding is that anyone should at a glance see what you wrote and understand it.

> Yet, "SUNNI_BERBER_COMPANY" for a VALENTIANIAN HOLY ORDER is okay with you? That confuses people too! You are a hypocrite!

Just because you scream it doesn't become more true. It replaces vanilla code. Very simple. We can add a hashtag if you want to.